docs/data/destinations/braze.md

```diff
@@ -9,17 +9,16 @@ Amplitude Data's Braze integration lets you stream your Amplitude event data str
 
 !!!note "Other Amplitude + Braze Integrations"
 
-    This integration streams Amplitude events to Braze. Amplitude offers other integrations with Braze: 
+    This integration streams Amplitude events to Braze. Amplitude offers other integrations with Braze:
 
     - [Send Cohorts to Braze](/data/destinations/braze-cohort)
     - [Braze Event Import](/data/sources/braze)
 
 ## Considerations
 
-- You must have a Braze account. 
-- Amplitude uses the `user_id` to match event data to a user within Braze. If a user with that `user_id` doesn't exist within Braze, Braze creates a user with that ID.
-- Amplitude sends the `user_id`, `event_name`, and `created_at` to Braze. 
-- Amplitude sends all user, group, and event properties as Braze custom event properties.
+- You must have a Braze account.
```

Comment:
Sadly, it's usually necessary. Some folks have a hard time understanding they need an account with whoever the integration is with. Because this one is so basic, we can probably get rid of it. If anyone complains we can add it back.